week-8/Homework/mandatory/2-fetch-exercise/exercise.js

```diff
@@ -17,10 +17,12 @@ Open index.html in your browser. Every time you refresh the page,
 a different greeting should be displayed in the box.
 */
 
-fetch('*** Write the API address here ***')
-    .then(function(response) {
-        return response.text();
-    })
-    .then(function(greeting) {
-        // Write the code to display the greeting text here
-    });
\ No newline at end of file
+fetch("https://codeyourfuture.herokuapp.com/api/greetings")
+  .then(function (response) {
+    return response.text();
+  })
+  .then(function (greeting) {
+    // Write the code to display the greeting text here
+    let paragraph = document.getElementById(`greeting-text`);
```

Comment:
Looks good. 

Minor point: you can use normal single quotes ('') rather than `` for your strings when it does not container variables within it.